UnameMeter.c

```diff
@@ -0,0 +1,57 @@
+/*
+htop - UnameMeter.c
+(C) 2004-2011 Hisham H. Muhammad
+Released under the GNU GPL, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "UnameMeter.h"
+
+#include "CRT.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <sys/utsname.h>
+
+/*{
+#include "Meter.h"
+}*/
+
+int UnameMeter_attributes[] = {
+   HOSTNAME
+};
+
+static void UnameMeter_updateValues(Meter* this, char* buffer, int size) {
+   (void) this;
+   struct utsname *data = NULL;
+   data = xMalloc(sizeof(struct utsname));
```

Comment:
@GiedriusS 
Hmm... I need to rethink of this a little bit. My original intention was to have this buffer allocated only once and uname() called only once. But the current Meter object does not allow persistent* storage of non-numeric values yet, so I changed to the allocate-and-call-uname-every-time approach.

So you are suggesting me to allocate this utsname buffer in the stack rather than in the heap, right? That sounds like a good idea, but I doubt whether the code will be portable (you know, `sizeof(struct utsname)` is implementation-defined and it could be quite large, and more likely to cause "overflow" for some systems).

* Persistent within the Meter object's lifetime